Tools/setup/ubuntu.sh

```diff
@@ -3,6 +3,18 @@
 INSTALL_NUTTX="true"
 INSTALL_SIM="true"
 
+# Initial messages
+echo
+echo -e "\033[1mWELCOME TO PX4. BUCKLE UP, WE ARE ABOUT TO TAKEOFF...\033[0m"
+echo
+echo -e "\033[0;32m\033[1mINSTALLING PX4 TOOLCHAIN!\033[0m"
+echo
+echo -e "\033[94m\033[1mThis could take a while (10min) make sure you go grab a cofee \033[33m:)\033[0m"
+echo
+
+# Sleep 5 seconds to show the initial messages
+sleep 5
```

Comment:
Let's not sleep unnecessarily. The intention of the script is that it's cheap and convenient (and safe) to re-run periodically to check for any updates.

Reply:
@TheLegendaryJedi can you please address here?

Reply:
I think we can close this, like @dagar said we are not supposed to introduce delays on the script. And without it the messages will not appear because they will be overwritten by the install messages.